### DISCUSSION TITLE
Problems running tests during build

### DISCUSSION THREAD

**Discussion by luca-bassoricci** [quarkusio/quarkus#24293] [Q&A]
Which is the right way to change port for test during build?
I have McAfeee installed on my pc aand it is listening on port 8081, so I have to change them during build passing `-Dquarkus.http.test-port=8888` to Maven but I have this problem:
```
RandomPortTest.testActualPortAccessibleToApp:42 1 expectation failed.
Response body doesn't match expectation.
Expected: "8888"
  Actual: 8080
```
Can't change McAfee properties due to policy restrictions.
Any advice?

**Comment by famod** [comment]
How does that test look like?

Btw, you could try setting a truly random port via `=0`.
